client/src/reducers/index.js

```diff
@@ -1,6 +1,8 @@
-import { combineReducers } from 'redux';
-import categoryList from './category_list';
+import { combineReducers } from "redux";
+import categoryList from "./category_list";
+import categoryReducer from "./categoryReducer";
 
 export default combineReducers({
-  categories: categoryList
+  categories: categoryList,
+  category: categoryReducer
```

Comment:
so correct me if im wrong but this part of the will just hold a single category correct?

Reply:
On line 7, yes